libraries/AP_CANManager/AP_CANManager.h

```diff
@@ -104,10 +104,10 @@ class AP_CANManager
     void log_retrieve(ExpandingString &str) const;
 
     // return driver type index i
-    Driver_Type get_driver_type(uint8_t i) const
+    Driver_Type get_driver_type(uint8_t i, bool want_param_value = false) const
```

Comment:
WTH is this new parameter?  Please extend the comment!
.

Reply:
fair enough, I'll add a comment to explain the weirdness.

Reply:
I've reworked this so it's no longer needed